app/forms/state_file/ny201_form.rb

```diff
@@ -27,6 +27,8 @@ class Ny201Form < QuestionsForm
                        :primary_signature,
                        :spouse_signature
 
+    validates :routing_number, length: { is: 9 }, allow_blank: true, routing_number: true
```

Comment:
add validation so we don't get bunk routing numbers, but still allow it to be blank